Assets/MixedRealityToolkit.Services/InputAnimation/Editor/InputRecordingWindow.cs

```diff
@@ -0,0 +1,330 @@
+            }
+            if (SimService == null)
+            {
+                EditorGUILayout.HelpBox("No input simulation service found", MessageType.Info);
+                return;
+            }
+
+            DrawAnimationInfo();
+
+            using (new GUILayout.HorizontalScope())
+            {
+                if (GUILayout.Button("Load ..."))
+                {
+                    string filepath = EditorUtility.OpenFilePanel(
+                        "Select input animation file",
+                        "",
+                        InputAnimationSerializationUtils.Extension);
+
+                    LoadAnimation(filepath);
+                }
+            }
+
+            using (new GUIEnabledWrapper(PlaybackService != null))
+            {
+                bool wasPlaying = PlaybackService.IsPlaying;
+
+                bool play, stepFwd, jumpBack;
+                using (new GUILayout.HorizontalScope())
+                {
+                    jumpBack = GUILayout.Button(new GUIContent(iconJumpBack, "Reset input animation"), "Button");
+                    play = GUILayout.Toggle(wasPlaying, wasPlaying ? new GUIContent(iconPlay, "Stop playing input animation") : new GUIContent(iconPlay, "Play back input animation"), "Button");
+                    stepFwd = GUILayout.Button(new GUIContent(iconStepFwd, "Step forward one frame"), "Button");
+                }
+
+                float time = PlaybackService.LocalTime;
+                float duration = (animation != null ? animation.Duration : 0.0f);
+                float newTime = GUILayout.HorizontalSlider(time, 0.0f, duration);
+
+                if (play != wasPlaying)
+                {
+                    if (play)
+                    {
+                        PlaybackService.Play();
+                    }
+                    else
+                    {
+                        PlaybackService.Pause();
+                    }
+                }
+                if (jumpBack)
+                {
+                    PlaybackService.LocalTime = 0.0f;
+                }
+                if (stepFwd)
+                {
+                    PlaybackService.LocalTime += Time.deltaTime;
+                }
+                if (newTime != time)
+                {
+                    PlaybackService.LocalTime = newTime;
+                }
+
+                // Repaint while playing to update the timeline
+                if (PlaybackService.IsPlaying)
+                {
+                    Repaint();
+                }
+            }
+        }
+
+        private void DrawAnimationInfo()
+        {
+            using (new GUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                GUILayout.Label("Animation Info:", EditorStyles.boldLabel);
+
+                if (animation != null)
+                {
+                    GUILayout.Label($"File Path: {loadedFilePath}");
+                    GUILayout.Label($"Duration: {animation.Duration}");
+                }
```

Comment:
Nit: Maybe if there's no animation, maybe worth showing a little label saying "no animation loaded" - otherwise I think you just have this trailing label that says "Animation Info:"